### ISSUE TITLE
Site logo link not generated correctly

### ISSUE THREAD

**Issue by michimani** [AngeloStavrow/indigo#34]
**Describe the bug**
Site logo link not generated correctly at page header.

**To Reproduce**
1. set `baseURL = "https://example.com/"` to `config.toml`
2. `hugo` or `hugo server -D` 
3. check the html source of site-logo part
4. incorrect link `https://example.com//images/site-logo.svg`

**Expected behavior**
Fix site-logo path in layout files.
- `layouts/404.html`
- `layouts/partials/header.html`

**Screenshots**
<img width="1228" alt="2018-10-24_17 10 54" src="https://user-images.githubusercontent.com/9986092/47416099-ecfc3380-d7af-11e8-91f9-7864f8627d61.png">

**Desktop (please complete the following information):**
 - OS: macOS 10.12.6
 - Browser Chrome
 - Version 70.0.3538.67（Official Build）

**Smartphone (please complete the following information):**
N/A

**Additional context**

**Comment by AngeloStavrow** [comment]
Good catch. Other assets are set up as `{{ .Site.BaseURL }}directory/asset-name.extension``; the site logo should be retrieved the same way as all other assets.

**Comment by michimani** [comment]
Thank you for your comment.
I have already prepared PR for solving this issue. May I create a PR?

**Comment by AngeloStavrow** [comment]
@michimani Yes, that would be wonderful! Thanks. 😊
